common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1187,6 +1193,29 @@ private AuthenticationConstants() {
         public static final String AZA_SCOPE = "aza";
     }
 
+    /**
+     * Represents Broker operations that should be invoked by Authenticator.java (MSAL-Broker AccountManager flow).
```

Comment:
Why is this class called `MsalAccountManagerOperation` when the javadoc says that it represents Broker Operations?
May be rename to `BrokerAccountManagerOperation`?